apps/charterafrica/src/payload/collections/Fellowships.js

```diff
@@ -0,0 +1,97 @@
+import content from "../fields/content";
+import richText from "../fields/richText";
+import slug from "../fields/slug";
+
+const Fellowships = {
+  slug: "fellowships",
+  admin: {
+    useAsTitle: "title",
+    defaultColumns: ["title", "author", "publishedOn"],
+  },
+  access: {
+    read: () => true,
+  },
+  versions: {
+    drafts: true,
+  },
+  fields: [
+    {
+      name: "title",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      type: "text",
+      localized: true,
+      required: true,
+    },
+    slug(),
+    {
+      name: "coverImage",
+      label: {
+        en: "Cover Image",
+        pt: "Imagem de capa",
+        fr: "Image de couverture",
+      },
+      type: "upload",
+      relationTo: "media",
+      required: true,
+      filterOptions: {
+        mimeType: { contains: "image" },
+      },
+    },
+    {
+      name: "category",
+      label: {
+        en: "Category",
+        fr: "Catégorie",
+        pt: "Categoria",
+      },
+      required: true,
+      type: "select",
+      options: [
+        {
+          label: {
+            en: "Technologies",
+            fr: "les technologies",
+            pt: "Tecnologias",
+          },
+          value: "technologies",
+        },
+        {
+          label: {
+            en: "Other",
+            pt: "Outro",
+            fr: "Autre",
+          },
+          value: "other",
+        },
```

Comment:
Cool, but lets also confirm if these are the only 2 categories we should support.